.github/workflows/check-dist.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         run: npm ci
 
       - name: Rebuild the dist/ directory
-        run: npm run build
+        run: npm run release
```

Comment:
FYI: this was not working properly, the build npm script did not change the `dist/` contents. I fixed it so it's properly calling release now to ncc the new js